update the tilt doc with screenshots (#1633)
kubernetes-sigs/cluster-api-provider-azure: templates/flavors/README.md

    }
}
```

> After having the tilt-settings.json, follow these two steps to deploy a workload cluster

Reviewer: ```suggestion
After updating tilt-settings.json, follow these two steps to deploy a workload cluster:
```

> 1. ``tilt up`` in the root of cluster-api-provider-azure repo. Note that the tilt-settings.json also resides here in the 

Reviewer: ```suggestion
1. Run ``make tilt-up`` in the root of cluster-api-provider-azure repo. Note that the tilt-settings.json also resides here in the 
```

> root of this repo. After this, press `space` to open the tilt UI in browser. See the following example:

Reviewer: ```suggestion
root of this repo. After tilt has initialized, press `space` to open the tilt web UI in a browser. See the following example:
```
